src/components/vehicles/smarteq/vehicle.vue

```diff
@@ -14,16 +14,37 @@
 			</template>
 		</openwb-base-text-input>
 		<openwb-base-text-input
+			v-model="password"
```

Comment:
```suggestion
```
Das wird bereits in den Zeilen 22 und 23 erledigt.

Reply:
Ist geändert